.travis.yml

```diff
@@ -21,6 +23,11 @@ before_install:
   - "gem update bundler"
   - "[ -f /tmp/beanstalkd-1.10/Makefile ] || (curl -L https://github.com/kr/beanstalkd/archive/v1.10.tar.gz | tar xz -C /tmp)"
   - "pushd /tmp/beanstalkd-1.10 && make && (./beanstalkd &); popd"
+  - "[[ $GEM != 'av:ujs' ]] || nvm install node"
+  - "[[ $GEM != 'av:ujs' ]] || node --version"
+  - "[[ $GEM != 'av:ujs' ]] || (pushd actionview && npm install; popd)"
```

Comment:
```
- "[[ $GEM != 'av:ujs' ]] || (cd actionview && npm install)"
```

The parens give us a subshell, so the cwd change is already isolated. Seems clearer to me, but up to you.